Wrappers/Python/cil/utilities/jupyter/__init__.py

```diff
@@ -164,18 +164,24 @@ def islicer(data, direction=0, title="", slice_number=None, cmap='gray',
     if slice_number is None:
         slice_number = int(data.shape[direction]/2)
 
-    style = {'description_width': '130px'}
-    height_layout = widgets.Layout(height='20px')
+    if title==None:
+        title = "Direction {}: Slice ".format(axis_labels[direction])
```

Comment:
```suggestion
        title = "Direction {}: Slice".format(axis_labels[direction])
```

(the title is formatted with a space elsewhere, so we should remove this unless you intended to have two spaces)